### PR TITLE
Feature/image upload

### PR DESCRIPTION
**Pull request by pranay2401**
# CHANGELOG
* Added support to delete or unpublish a movie.
* Added support to upload or delete thumbnail or background image for a movie.

### CODE REVIEW

File: packages/im-studio/src/organisms/publishTab/PublishTab.js [hunk 1]
```diff
@@ -44,6 +44,21 @@ const PublishTab = (props) => {
...
   };
 
+  const onUnpublish = (values) => {
+    dispatch(
+      updateMovieByID({
+        ...tabdata,
+        isPublished: false,
+      })
+    );
+    history.push("/dashboard");
```
Comment (ukirderohit) [inline packages/im-studio/src/organisms/publishTab/PublishTab.js]: please use url utility

File: packages/im-studio/src/organisms/publishTab/PublishTab.js [hunk 2]
```diff
@@ -44,6 +44,21 @@ const PublishTab = (props) => {
...
+  };
+
+  const onDelete = (values) => {
+    dispatch(deleteMovie({ movieId: id }, history));
+    history.push("/dashboard");
```
Comment (ukirderohit) [inline packages/im-studio/src/organisms/publishTab/PublishTab.js]: same

**Review by ukirderohit — CHANGES_REQUESTED** [review]

File: packages/im-studio/src/organisms/publishTab/PublishTab.js (hunk 2 above)
Comment (pranay2401) [reply packages/im-studio/src/organisms/publishTab/PublishTab.js]: Done

File: packages/im-studio/src/organisms/publishTab/PublishTab.js (hunk 1 above)
Comment (pranay2401) [reply packages/im-studio/src/organisms/publishTab/PublishTab.js]: Done

**Review by ukirderohit — APPROVED** [review]

**Pull request merged**